doc/image-from-registry.md

```diff
@@ -2,47 +2,50 @@
 The purpose of this document is to show how to create registry image containing a Virtual Machine image that can be imported into a PV.
 
 ## Prerequisites
-The VM disk image should be called 'disk.img' and placed in the root folder.
+Import from registry should be able to consume the same container images as containerDisk (https://github.com/kubevirt/kubevirt/blob/master/docs/container-register-disks.md).
+Thus the VM disk image file to be consumed must be located under /disk directory in the container image. The file can be in any of the supported formats : qcow2, raw, archived. There is no any special naming contraints for the VM disk file.
 
-## Create an image with Buildah
+## Import from existing containerDisk images from kubevirt repository 
+For example vmidisks/fedora25:latest as described in https://github.com/kubevirt/kubevirt/blob/master/docs/container-register-disks.md
 
+## Create a container image with Buildah
 Buildah is a tool that facilitates building Open Container Initiative (OCI) container images.
-
 More information is available here: [Buildah tutorial](https://github.com/containers/buildah/blob/master/docs/tutorials/02-registries-repositories.md).
 
-Create a new container image from scratch and keep its name in a variable:
+Create a new directory /tmp/vmdisk with the following Docker file and a vm image file (ex: fedora28.qcow2)
+Create a new container image with the following docker file 
 
 ```bash
-$ newcontainer=$(buildah from scratch)
+cat << END > Dockerfile
+FROM kubevirt/container-disk-v1alpha
+ADD fedora28.qcow2 /disk
+END
 ```
-
-Add the VM disk to the container image and commit:
+Build and push image to a registry. 
+Note: In development environment you can push to 
+1. A cluster local cdi-docker-registry-host which hosts docker registry and is accessible within the cluster via cdi-docker-registry-host.kube-system. The registry is initialized from cluster-sync flow and is used for functional tests purposes. 
```

Comment:
`cdi-docker-registry-host`, `cdi-docker-registry-host.kube-system` and `cluster-sync` inside backticks.